addons/web/static/src/js/views/form/form_controller.js

```diff
@@ -218,7 +235,32 @@ var FormController = BasicController.extend({
     //--------------------------------------------------------------------------
     // Private
     //--------------------------------------------------------------------------
-
+    /**
+     * Assign on the buttons save and discard additionnal behavior to facilitate the work of the users doing input only using the keyboard
+     * 
+     * @param {jQueryElement} $saveCancelButtonContainer  The div containing the save and cancel buttons
+     */
+    _assignSaveCancelKeyboardBehavior:function($saveCancelButtonContainer) {
```

Comment:
should be tagged private.  also, not really important, but everywhere else, we add a space after the : and after function

Reply:
done